slides.md

```diff
@@ -156,57 +156,97 @@ import anotherFunction, {aFunction} from "./someFile";
 
 ---
 # Hello Todo!
```

Comment:
looks like a typo, should be `Hello Toto` who is a good target audience because I'm sure he would love to learn more about JS cc/ @mr-archano :trollface:

Reply:
I mean ActionScript and JS are basically the same thing, right? :trollface:

Reply:
It's all ECMA script stuff anyway @rock3r. Also: :x: until that title is changed, you set some expectations now.